### PR TITLE
Alibaba cloud type and updating pre commit versioning  

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Adding alibaba specific node scenarios 

I also ran into issues with pre-commit versioning that were fixed with the below updates

### Fixes
https://github.com/cloud-bulldozer/kraken/issues/202

### CODE REVIEW

**Review by yogananth-subramanian — APPROVED** [review]
LGTM!
It errored out appropriately when I tried executing with fake alibaba credentials.

**Review by mffiedler — APPROVED** [review]
Tested on an alicloud cluster.   Cloud-specific scenarios all work well.

**Review by chaitanyaenr — APPROVED** [review]
Code looks good to me! I didn't test it due to lack of access to Alibaba cloud.

File: kraken/node_actions/abstract_node_scenarios.py [hunk 1]
```diff
@@ -7,17 +7,22 @@
...
-    def node_stop_scenario(self, instance_kill_count, node, timeout):
+    def node_stop_scenario(self, instance_kill_count, node, timeout, force_action=False):
         pass
 
     # Node scenario to stop and then start the node
-    def node_stop_start_scenario(self, instance_kill_count, node, timeout):
+    def node_stop_start_scenario(self, instance_kill_count, node, timeout, force_action=False):
         logging.info("Starting node_stop_start_scenario injection")
-        self.node_stop_scenario(instance_kill_count, node, timeout)
+        self.node_stop_scenario(instance_kill_count, node, timeout, force_action)
```
Comment (paigerube14) [inline kraken/node_actions/abstract_node_scenarios.py:25]: @chaitanyaenr @mffiedler any ideas on how to rework this so I don't have to go through and add "force_action" as a parameter in all the stop scenario function definitons in each cloud? Want to add in the option for alibaba specifically and ignore for all other clouds

Works for alibaba but get the below error on AWS. 
`TypeError: node_stop_scenario() takes 4 positional arguments but 5 were given`

Comment (chaitanyaenr) [reply kraken/node_actions/abstract_node_scenarios.py:25]: I would go with not defining the force_action as parameter to keep things generic for all cloud providers and instead set force_action=true by default for Alibaba node actions as that might be more representative of chaos. Thoughts?

Comment (paigerube14) [reply kraken/node_actions/abstract_node_scenarios.py]: Set default force value to True

**Pull request merged**